tutorial/hextof_config.yaml

```diff
@@ -0,0 +1,151 @@
+core:
+  loader: flash
+  beamtime_id: 11019101
+  year: 2023
+  beamline: pg2
+  instrument: hextof
+  paths:
+    data_raw_dir: "/asap3/flash/gpfs/pg2/2023/data/11019101/raw/hdf/offline/fl1user3"
```

Comment:
With this path, this config is only valid for desy cluster. We can provide that as a note, especially in the tutorial notebook (and also that this should be changed if the files are stored locally).

Reply:
if you have access to the data, you have access to maxwell...

Reply:
So far test data is sitting there, so I would structure tutorials based on that.
If we make data available outside for download, we can then generalize.